lib/tty/prompt/answers_collector.rb

```diff
@@ -9,6 +9,7 @@ class AnswersCollector
       def initialize(prompt, options = {})
         @prompt  = prompt
         @answers = options.fetch(:answers) { {} }
+        @multiple = false
```

Comment:
That doesn't seem right. This flag sets a state for the whole collector. What if I have many keys with multiple values etc...